Formula/s/stencil.rb

```diff
@@ -1,10 +1,11 @@
 class Stencil < Formula
-  desc "Smart templating engine for service development"
-  homepage "https://engineering.outreach.io/stencil/"
-  url "https://github.com/getoutreach/stencil/archive/refs/tags/v1.40.0.tar.gz"
-  sha256 "4d7ae67613a5dc6e710f551a528fa136ecdcad7bb758d47d542f0e8b5692b7cb"
+  desc "Modern living-template engine for evolving repositories"
+  homepage "https://stencil.rgst.io"
+  url "https://github.com/rgst-io/stencil.git",
+      tag:      "v2.0.1",
+      revision: "ccc083c7ff1a54623e99c59e9bcab1008c189d6f"
```

Comment:
We try to avoid git checkouts unless absolutely necessary. Are we not able to build with a tarball?

Reply:
To be honest, I was following [goreleaser's Formula](https://github.com/Homebrew/homebrew-core/blob/master/Formula/g/goreleaser.rb) here since they both use the same version library and have similar build processes. Does [getting the commit hash](https://github.com/Homebrew/homebrew-core/pull/203360/files#diff-a05f4bb0d8428da5c222ada3f99a5e9cb14976433a56ecb328f18a0e287a4d08R25) work with a tarball? If so, it should build fine and I can switch it!

Reply:
No, we can't determine the commit hash from a tarball. We typically eschew having the commit hash via checkout in favor of tarballs, unless there's a compelling reason to build from a checkout. We usually fill in the commit hash with "Homebrew" or leave it blank: https://github.com/Homebrew/homebrew-core/blob/4cfcf247356632cec65981550ec479a5c379df1a/Formula/i/ipsw.rb#L29

Reply:
Makes sense. I'll update this!

Reply:
Updated!